### PR TITLE
Added a Gaussian filter

### PR DESCRIPTION
**Pull request by LijieTu**
Added A Gaussian Filter

### CODE REVIEW

**Comment by yijiang1** [comment]
@LijieTu is a student at Cornell. He'll add some useful python data transforms.

File: tomviz/AddPythonTransformReaction.cxx [hunk 1]
```diff
@@ -414,6 +414,41 @@ OperatorPython* AddPythonTransformReaction::addExpression(DataSource* source)
...
+     connect(buttons, SIGNAL(accepted()), &dialog, SLOT(accept()));
+     connect(buttons, SIGNAL(rejected()), &dialog, SLOT(reject()));
+     v->addLayout(layout);
+     v->addWidget(buttons);
+     dialog.setLayout(v);
+     dialog.layout()->setSizeConstraint(QLayout::SetFixedSize); //Make the UI non-resizeable
+
+     if (dialog.exec() == QDialog::Accepted)
+     {
+         QMap<QString, QString> substitutions;
```
Comment (cryos) [inline tomviz/AddPythonTransformReaction.cxx]: This block is over indented - just two spaces.

**Comment by cryos** [comment]
Really minor whitespace, looks good to me otherwise.

**Comment by mathturtle** [comment]
@LijieTu It looks like someone else has merged something that modifies the same parts of the file AddPythonTransformReaction.cxx that you did.  You will need to merge your branch with master and resolve the conflicts before your branch can go in.  First make sure you have the latest version of the master branch:
```
git checkout master
git pull
```
Then check out your branch and run
```
git merge master
```
You will get some output that is essentially telling you that git was able to figure out how to merge all the changes except for that one file.  You will have to merge that file yourself.  Git will put sections like this in the file:
```
<<<<<<<<<<<
Code from your branch
============
Code that is there on the master branch
>>>>>>>>>>>>
```
You need to find each of these and fix them so that the code does what you want it to and compiles again.  I looked at the conflicts and it shouldn't be hard.  Then tell git you have fixed the conflicts by running
```
git add tomviz/AddPythonTransformReaction.cxx
```
Next commit the merged version of the code with `git commit`.  It will pre-fill the commit message with a message about the merge, which you can edit if you with.  Then push your new version here.

**Comment by LijieTu** [comment]
I guess it should be fine now? @mathturtle @yijiang1

File: tomviz/AddPythonTransformReaction.cxx [hunk 2]
```diff
@@ -379,6 +379,10 @@ OperatorPython* AddPythonTransformReaction::addExpression(DataSource* source)
       addPythonOperator(source, this->scriptLabel, this->scriptSource, substitutions);
     }
   }
+<<<<<<< HEAD
+  
+=======
+>>>>>>> master
```
Comment (yijiang1) [inline tomviz/AddPythonTransformReaction.cxx]: @LijieTu You need to delete the comments like line 382 to 385.

Comment (LijieTu) [inline tomviz/AddPythonTransformReaction.cxx]: done

**Pull request merged**